doc/development/guide/pullrequests.rst

```diff
@@ -1,74 +1,58 @@
 
-  .. code-block:: bash
+* **Documentation**: The "Documentation check" determines if there are any issues with the build, but you also need to inspect the relevant pages in the website build visually. You can access the temporary website by clicking on ``Details`` for ``docs/readthedocs.org:pennylane``, as shown in the following image.
 
-      pylint pennylane/path/to/modified/file.py
+.. image:: view_doc_build.jpeg
+    :width: 400px
+    :align: center
 
-Submitting to the PennyLane repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Ready for review?
+-----------------
 
-When ready, submit your fork as a `pull request <https://help.github.com/articles/about-pull-requests>`_
-to the PennyLane repository, filling out the pull request template. This template is added
-automatically to the comment box when you create a new issue.
+A review-ready pull request (PR) includes the following:
 
-* When describing the pull request, please include as much detail as possible
-  regarding the changes made/new features added/performance improvements. If including any
-  bug fixes, mention the issue numbers associated with the bugs.
+* **All new code is clearly commented and documented.**  See our :doc:`documentation guidelines <documentation>` for more details.
 
-* Once you have submitted the pull request, three things will automatically occur:
+* **All changes must include tests.** If you fix a bug, write a test that would fail before the bugfix. See :doc:`Tests <tests>` for more information.
 
-  - The **test suite** will automatically run on `GitHub Actions <https://github.com/PennyLaneAI/pennylane/actions>`_
-    to ensure that all tests continue to pass.
+* **Code conforms to PEP8 standards.** As mentioned in the section above, all code needs to be formatted via `Black <https://black.readthedocs.io/en/stable/>`_ with a line length of 100.
 
-  - Once the test suite is finished, a **code coverage report** will be generated on
-    `Codecov <https://codecov.io/gh/PennyLaneAI/pennylane>`_. This will calculate the percentage
-    of PennyLane covered by the test suite, to ensure that all new code additions
-    are adequately tested.
+* **Write a changelog entry for all Pull Requests.** The changelog entry should include a link back to the PR. More significant user-facing changes should include a code example. In addition to the changelog entry itself, add your name to the alphabetical contributors' list at the bottom of each release's section.
 
-  - Finally, the **code quality** is calculated by
-    `Codefactor <https://www.codefactor.io/repository/github/pennylaneai/pennylane>`_,
-    to ensure all new code additions adhere to our code quality standards.
+Stuck? You can make a "Work in Progress" PR with ``[WIP]`` in the title or with the "WIP" label.  That way, we can see your work and give feedback early.
```

Comment:
It might be worth moving this up on the page, to indicate early on that this is an option.